### PR TITLE
Enable Live RadioSchedule data on Test env

### PR DESCRIPTION
**Pull request by sareh**
Part of #5854

**Overall change:** 
Enabling using live data for radio schedules on the test environment.
(Note that localhost can already use the local json in the `/data/` directory.)

**Code changes:**
- Update data files to reflect routes on Test and Live
- Move endpoint logic to separate endpoint.js file
- Enable overrides via `renderer_env=live` on non-live builds

Testing - cannot currently be tested locally - should be tested after merging.
~- Follow build instructions here for building the application for the TEST environment https://github.com/bbc/simorgh#using-environment-builds-locally (note that you will also need to update these in the .env file before running `npm run start`, to enable local javascript bundles to be used.~
```
SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN=http://localhost:7080
SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/
LOG_DIR='log'
```
~- Then visit a front page for a service that has a radio schedule http://localhost:7080/persian?renderer_env=live~
~- See data being used from the live environment for radio schedules.~

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/utilities/getRadioSchedulesUrls/index.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+export const getRadioScheduleEndpoint = ({
+  service,
+  radioService = service,
+  env,
+  queryString,
+}) => {
+  if (env !== 'live' && queryString) {
+    return `/${service}/bbc_${radioService}_radio/schedule.json${queryString}`;
+  }
+  return `/${service}/bbc_${radioService}_radio/schedule.json`;
```
Comment (DenisHdz) [inline src/app/lib/utilities/getRadioSchedulesUrls/index.jsx]:
```
const query = env !== 'live' && queryString ? queryString : '';
return `/${service}/bbc_${radioService}_radio/schedule.json${query}`;
```

Comment (sareh) [reply src/app/lib/utilities/getRadioSchedulesUrls/index.jsx]: @DenisHdz Thanks, I've done this now.

**Review by DenisHdz — COMMENTED** [review]
@sareh Should we expect to see radio schedules live data for Pidgin as per the description?

**Comment by sareh** [comment]
@DenisHdz The pidgin link was incorrect. The correct example would be this: http://localhost:7080/persian?renderer_env=live but I believe it is not working on local dev, likely since the client-side javascript is not rendering at all for the test build. I will pull this back into in-progress to resolve this issue.

**Comment by sareh** [comment]
@DenisHdz As a result of discussions with the team, we've decided to merge this in (after code reviews) and test on the Test environment, since this other issue (https://github.com/bbc/simorgh/issues/5947) is a longer-term fix that may take some time.

File: src/app/lib/utilities/getRadioSchedulesUrls/index.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export const getRadioScheduleEndpoint = ({
+  service,
+  radioService = service,
+  env,
+  queryString,
+}) => {
+  const query = env !== 'live' && queryString ? queryString : '';
```
Comment (sareh) [inline src/app/lib/utilities/getRadioSchedulesUrls/index.jsx:7]: When you're not on the live environment, the query string is added.

**Review by DenisHdz — APPROVED** [review]
Great work both 👍

**Review by tochwill — APPROVED** [review]

**Comment by tochwill** [comment]
Thank you for this!

**Pull request merged**